src/semantic_world/world.py

```diff
@@ -1322,6 +1369,54 @@ def compute_split_chain_of_connections(
             tip_connections.append(self.get_connection(tip_chain[i], tip_chain[i + 1]))
         return root_connections, tip_connections
 
+    @property
+    def kinematic_structure_entities_topologically_sorted(
+        self,
+    ) -> List[KinematicStructureEntity]:
+        indices = rx.topological_sort(self.kinematic_structure)
+
+        return [self.kinematic_structure[index] for index in indices]
+
+    @property
+    def bodies_topologically_sorted(self) -> List[Body]:
+        return [
+            entity
+            for entity in self.kinematic_structure_entities_topologically_sorted
+            if isinstance(entity, Body)
+        ]
+
+    def move_subgraph_from_root_to_new_world(
```

Comment:
I think this name is confusing since the use of "root" is ambiguous, "copy_subgraph_to_new_world" is better